ui/src/pages/publications/[id]/index.tsx

```diff
@@ -184,7 +184,7 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
 
     const user = Stores.useAuthStore((state: Types.AuthStoreType) => state.user);
     const isBookmarkButtonVisible = useMemo(() => {
-        if (!user || !publicationVersion) {
+        if (!user || !publicationVersion || (publicationVersion && publicationVersion.currentStatus !== 'LIVE')) {
```

Comment:
I think we can simplify this useMemo by simply returning `user && publicationVersion?.currentStatus === 'LIVE'`

Reply:
Yeah, that's true. Now fixed